tests/Application/package.json.~1.12.0.dist

```diff
@@ -0,0 +1,58 @@
+{
```

Comment:
The documentation update is necessary for the file-split.

Reply:
Removed file split